rfcs/text/0000-authentication-fields.md

```diff
@@ -0,0 +1,155 @@
+# 0000: Authentication Fields RFC
+
+Stage: 0
+Date: 2021-08-08
+
+This RFC will provide normalization for fields related to authentication fields to assure that they are retained primarily in ECS core and in any extended fieldset when needed. These fields are important to normalize into ECS fields in order to maximize effectiveness of cross log utilization. 
+
+## Fields
+
+#### Activity Log
+|Proposed Field Name|Type|Value|Azure Field|
+| --- | --- | --- | --- |
+|authentication.scope|keyword|user_impersonation|azure.activitylogs.identity.claims.http://schemas_microsoft_com/identity/claims/scope|
+|authentication.authentication.requirement|keyword|multiFactorAuthentication|azure.activitylogs.properties.authenticationRequirement|
+|authentication.additional.details|keyword|MFA requirement satisfied by claim in the token|azure.activitylogs.properties.status.additionalDetails|
+|authentication.token.type|keyword|AzureAD|azure.activitylogs.properties.tokenIssuerType|
+|authentication.token.identifier|keyword|MjlkY2M0MzItNWU4YS00NjU5LTlmMDMtNmVkZTE4NDAwMzAw|azure.activitylogs.properties.uniqueTokenIdentifier|
+|authentication.authorization|keyword|ROOTMANAGESHAREDACCESSKEY|azure.resource.authorization_rule|
+|authentication.isInteractive|keyword|FALSE|azure.signinlogs.properties.is_interactive|
+|authentication.token|keyword|AzureAD|azure.signinlogs.properties.token_issuer_type|
+
+
+#### SignIn Log
+|Proposed Field Name|Type|Value|Azure Field|
+| --- | --- | --- | --- |
+|authentication.authentication.requirement|keyword|singleFactorAuthentication|azure.signinlogs.properties.authentication_requirement|
+|authentication.isInteractive|boolean|FALSE|azure.signinlogs.properties.is_interactive|
+|authentication.risk|keyword|low|azure.signinlogs.properties.risk_level_aggregated|
```

Comment:
would you intend to re-use the in progress [risk fieldset](https://github.com/elastic/ecs/blob/main/experimental/schemas/risk.yml) here?

Reply:
@kgeller if the risk fieldset would help, then that's fine. But the intent wasnt to necessarily use it.